grammar.js

```diff
@@ -1157,7 +1162,7 @@ module.exports = grammar({
         optional($.inline_modifier),
         field("value", $.expression),
         "match",
-        field("body", choice($.case_block, $.indented_cases)),
+        field("body", choice($.case_block, $.indented_cases, $.non_indented_cases)),
```

Comment:
Should `non_indented_cases` be aliased to `indented_cases` here to preserve compatibility for any queries that may not expect nodes other than `case_block`/`indented_cases`, wdyt?

Reply:
+1 to this.